content/en/_index.html

```diff
@@ -13,247 +13,1763 @@
-        Azure Code Signing service, Ratify, etc.
+
+  <div class="u-container-5">
+    <div class="u-f-cont-5">
+      <div class="u-f-cot-1">
+        <img src="cot5.png" class="cot-1" />
+      </div>
+      <h4 class="u-h-5">Community-driven</h4>
+      <p class="u-d-5">
+        Our platform is community-driven, with active participation and
+        collaboration from developers worldwide. Join our vibrant community to
+        share knowledge, contribute to the ecosystem, and benefit from
+        collective expertise to drive innovation and growth.
       </p>
+      <div class="u-sub-p-1">
+        <div>
+          <div class="sep-p">
+            <div class="check-img">
+              <img src="check.png" class="chk-1" />
+            </div>
+            <span class="sub-1">Fast iteration</span>
+          </div>
+          <div class="sep-p">
+            <div class="check-img">
+              <img src="check.png" class="chk-1" />
+            </div>
+            <span class="sub-1">Diverse expertise</span>
+          </div>
+        </div>
+        <div class="t-sub-1">
+          <div class="sep-p">
+            <div class="check-img">
+              <img src="check.png" class="chk-1" />
+            </div>
+            <span class="sub-1">Open-source & community-driven</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="u-f-img-5">
+      <img src="feature5.png" alt="" class="us-img-5" />
     </div>
   </div>
-  <div class="horizontalCard gap-md space-between is-align-items-center box flex">
-    <div class="img-container-md">
-      <img src="zot.svg" alt="Zot logo" />
+</div>
+
+{{% /blocks/lead %}} {{% blocks/lead color="white" %}} {{< blog/list >}} {{%
+/blocks/lead %}} {{% blocks/lead color="white" %}}
+<!-- Blogs -->
+
+<div class="main-ad">
+  <h1 class="title-at">Adopted and trusted by</h1>
+  <p class="width-60 is-centered font-size-md" id="desc-at">
+    Industry-leading enterprises and organizations are using the Notary Project
+    for research, production, and integration with security products. If you are
+    using the Notary Project, please share your case with us
+  </p>
+  <div
+    class="flex gap-md width-80 is-centered font-size-sm margin-md-top is-mobile-responsive fullwidth-tablet"
+    id="at-blocks"
+  >
+    <div class="horizontalCard flex gap-md is-align-items-center" id="s-block">
+      <div class="img-container-md">
+        <img src="aws_logo.svg" alt="Aqua logo" />
+      </div>
+      <div class="text-container text-left">
+        <p class="desc-at-block">
+          AWS team is using and contributing to Notation, building the
+          cryptographic signing services for customers
+        </p>
+      </div>
     </div>
-    <div class="text-container text-left">
-      <p>Zot registry supports store Notation signature as OCI artifacts</p>
+    <div
+      class="horizontalCard flex gap-md space-between is-align-items-center"
```

Comment:
The layout effect looks not correct on my 14' inch screen. 
![image](https://github.com/notaryproject/notaryproject.dev/assets/40452856/c2dbc47e-f67a-467f-9fcd-e8f59def5f0d)